Vagrantfile

```diff
@@ -129,6 +134,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     end
 
     app.vm.provision "ansible" do |ansible|
+      ansible.extra_vars = extra_vars
```

Comment:
On my machine...

`$HOSTNAME` is empty. Consider removing this.

`$USER` is set to `AZVA-INT\kdeloach`. May want to verify these are valid characters for resource names.